templates/legal/ubuntu-pro/personal/index.md

```diff
@@ -14,31 +14,31 @@ These Terms of Service cover the provision of Canonical's **Ubuntu Pro - persona
 
 You must be at least 13 years old to use the Service. If you are between age 13 and 18, you confirm that you have your parent's or legal guardian's consent to use the Services and that they have read and agreed to these Terms of Service.
 
 Canonical may update these Terms of Service from time to time in its discretion.
 
 ## 2. Service
 
-The Service provides users of Ubuntu with a free subscription to use the Service on up to five physical or virtual Ubuntu systems for the eligible Ubuntu LTS versions as designated by Canonical.
+The Service provides users of Ubuntu with a free subscription to use the Service on up to five physical Ubuntu systems with unlimited number of VMs or containers for the eligible Ubuntu LTS versions as designated by Canonical.
 
-Details about the Service can be found here:  https://ubuntu.com/legal/ubuntu-pro-description
+Details about the Service can be found here: https://ubuntu.com/legal/ubuntu-pro-description
 
-For recognised Ubuntu Community Members (as designated by Canonical) the Services can be used on up to fifty physical or virtual Ubuntu systems for the eligible Ubuntu LTS versions as designated by Canonical. 
+For recognised Ubuntu Community Members (as designated by Canonical) the Services can be used on up to fifty physical Ubuntu systems with unlimited number of VMs or containers for the eligible Ubuntu LTS versions as designated by Canonical.
 
 ## 3. Account
 
 You will need a Single Sign On Ubuntu One account to access and use the Service. If you do not have an account you will need to create one. You are responsible for choosing an appropriate password for your accounts and for keeping such password secure. Canonical will not ask you for your password and you should not reveal it to anyone. You are responsible for keeping your account details up to date.
 
-We reserve the right to reject your request for an account or to immediately cancel or suspend your account and your use of the Services at any time if you do not comply with the following requirements. You must not attempt to create an account or use the Service if doing so would violate these Terms of Service. You must not:            
+We reserve the right to reject your request for an account or to immediately cancel or suspend your account and your use of the Services at any time if you do not comply with the following requirements. You must not attempt to create an account or use the Service if doing so would violate these Terms of Service. You must not:
 
-- use the software made available through the Service for more than the entitled number of physical or virtual Ubuntu systems except as otherwise agreed, nor resell or attempt to resell the Services.
+- use the software made available through the Service for more than the entitled number Ubuntu systems except as otherwise agreed, nor resell or attempt to resell the Services.
```

Comment:
Is this a mistake in the copy doc? I feel like there should be an "of" here.

```suggestion
- use the software made available through the Service for more than the entitled number of Ubuntu systems except as otherwise agreed, nor resell or attempt to resell the Services.
```

Reply:
Yes, this makes total sense, I have  left a comment in the copydoc. Thanks

Reply:
They replied to your comment. I'm approving this PR, you can merge it after making the change.